Source/AzureSqlBulkFailover.md

```diff
@@ -0,0 +1,191 @@
+# AzureSqlBulkFailover.ps1
+
+This script is used to failover all databases and elastic pools in a subscription to a secondary, already upgraded replica. It filters by subscription, resource group and logical server name. It is intended to be used to facilitate CMW customers to upgrade their databases on demand when upgrades are ready (one touch).
+
+## Usage
+
+This script is intended to be run as an Azure Automation Runbook or locally.
+
+## Parameters
+
+- `SubscriptionId`: The ID of the subscription to failover. Optional.
+- `ResourceGroupName`: The name of the resource group to failover. Optional.
+- `LogicalServerName`: The name of the logical server to failover. Optional.
+
+## Notes
+
+- This script will failover ALL resources that the caller has access to in all subscriptions in the tenant, filtering by SubscriptionId, ResourceGroupName and LogicalServerName.
+- The base URI for ARM API calls is `https://management.azure.com`.
+- The script will wait for 15 seconds on each itteration before checking failover status.
+
+## Enumerations, globals and helper functions
+
+- `FailoverStatus`: An enumeration containing resource object FailoverStatus values.
+    - `Pending`: The resource is pending to be failedover.
+    - `InProgress`: The failover process is in progress.
+    - `Succeeded`: The failover process succeeded.
+    - `Skipped`: The failover process was skipped (the database did not need to be failed over or was not eligible).

```

Comment:
I don't see Skipped used... can it be removed from the script? (probably in a different PR, it's not related to your goals for this one)